GOVERNANCE.md

```diff
@@ -0,0 +1,124 @@
+  revenue.
+* Elect a Chair of the Diamond Bluff project to preside over meetings,
+  authorize expenditures approved by the budget, and manage any day-to-day
+  operations, and vote on decisions or matters before the Board of Directors.
+
+The Board of Directors shall::
+
+* Demonstrate plans and the means to coordinate with the open source project’s
+  developer community, including on topics such as branding, logos, and other
+  collateral that will represent the community.
+* Engage in a professional manner consistent with maintaining a cohesive
+  community, while also maintaining the goodwill and esteem of the project's
+  foundation, in the open source software community.
+  community.
+* Respect the rights of all trademark owners, including any branding and usage
+  guidelines.
+* Engage the project's foundation for all press and analyst relations activities.
+* Upon request, provide information regarding project participation, including
+  information regarding attendance at project-sponsored events, to the
+  project's foundation.
+* Engage the project's foundation for any Diamond Bluff project websites, and
+  operate under such rules and procedures as may from time to time be approved
+  by the Diamond Bluff Board of Directors and confirmed by the project's
+  foundation.
+
+### Technical Steering Committee
+
+The Diamond Bluff Technical Steering Committee shall be composed as follows:
+
+* One appointed representative from each Platinum member
+* A maintainer from each Diamond Bluff sub-project
+
+In addition, anyone can participate on the Technical Steering Committee by
+contributing to the technical community and becoming a maintainer of a
+sub-project.
+
+The responsibilities of the Diamond Bluff Technical Steering Committee are
+as follows:
+
+* Coordinating the technical direction of the Diamond Bluff Project
+* In coordination with the Board of Directors, approving new sub-projects for
+  creation under the Diamond Bluff organization.
+* Helping to identify maintainers of sub-projects, and ensure they have the
+  technical support necessary to be successful in their role as a maintainer.
+* Communicating with external and industry organizations concering Diamond
+  Bluff technical matters.
+* Appointing represenatives to work with other open source or open standards
+  communities.
+* Voting on technical matters relating to the code base of all sub-projects.
+
+## Voting
+
+While is the goal of the project to strive for consensus as a community, if any
+decision requires a vote to move forward, the Board of Directors members shall
+be entitled to vote on a one vote per representative basis.
+Decisions by vote shall be based on a majority vote only when sixty percent
+(>= 60$) or greater of the Board of Directors representatives are either
+present or particiapting electronically. In the event of a tied vote, the Chair
+shall be entitiled to submit a tie-breaking vote
```

Comment:
Add para:

All elections shall be held using an online preference election mechanism to be selected and approved by the Board.